contracts/src/Teleporter/SafeERC20TransferFrom.sol

```diff
@@ -29,10 +32,9 @@ library SafeERC20TransferFrom {
         erc20.safeTransferFrom(msg.sender, address(this), amount);
         uint256 balanceAfter = erc20.balanceOf(address(this));
 
-        require(
-            balanceAfter > balanceBefore,
-            "Balanced unchanged after safe transfer."
-        );
+        if (balanceAfter <= balanceBefore) {
+            revert BalanceUnchanged();
```

Comment:
Should we check for "==" instead given the error name?

Reply:
fixed using "=="

Reply:
Are we able to guarantee that the above call to `erc20.balanceOf` won't result in `balanceAfter < balancheBefore`? Despite the error name, checking `balanceAfter <= balanceBefore` seems safer.